vars/centos-8.yml

```diff
@@ -14,5 +14,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+_rpm_repository:
+  - name: "https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm"
+    state: "latest"
+    enablerepo: "epel-testing"
+
 _yum:
-  - { state: "latest", name: "firefox" }
+  - { state: "latest", name: "gnucash" }
```

Comment:
EPEL Testing have package for CentOS/RHEL 8, see https://centos.pkgs.org/8/epel-testing-x86_64/gnucash-4.1-1.el8.x86_64.rpm.html#howto

Need the `enablerepo` additional parameters

Reply:
I have followed pkgs's Install Howto. 
```yml
_rpm_repository:
  - name: "https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm"
    state: "latest"

_yum:
  - { state: "latest", name: "gnucash", enablerepo: "epel-testing"}
```
It pop out the error message `"No package gnucash available."`